programs/marginfi/src/instructions/marginfi_group/panic_unpause_permissionless.rs

```diff
@@ -0,0 +1,43 @@
+use anchor_lang::prelude::*;
+use marginfi_type_crate::{constants::FEE_STATE_SEED, types::FeeState};
+
+use crate::state::panic_state::PanicStateImpl;
+
+pub fn panic_unpause_permissionless(ctx: Context<PanicUnpausePermissionless>) -> Result<()> {
```

Comment:
Oh, so it has to be manually unpaused by someone, even if the 15 minutes run out. Does unpausing really require an instruction? It can't happen automatically?